src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int TEST_AGE = 22;
+    private static final String DEFAULT_LOGIN = "defaultLogin";
+    private static final String TEST_LOGIN = "testLogin";
+    private static final String TEST_LOGIN_1 = "testLogin1";
+    private static final String TEST_PASSWORD = "testPassword";
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        assertThrows(RegistrationException.class,
+                () -> service.register(new User(null, TEST_PASSWORD, TEST_AGE)));
+    }
+
+    @Test
+    void register_nullPassword_notOk() {
+        assertThrows(RegistrationException.class,
+                () -> service.register(new User(TEST_LOGIN, null, TEST_AGE)));
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        assertThrows(RegistrationException.class,
+                () -> service.register(new User(TEST_LOGIN, TEST_PASSWORD, null)));
+    }
+
+    @Test
+    void register_returnUser_ok() {
+        User expected = new User(TEST_LOGIN, TEST_PASSWORD, TEST_AGE);
+        User actual = service.register(expected);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_existingLogin_notOk() {
+        Storage.people.add(new User(DEFAULT_LOGIN, TEST_PASSWORD, TEST_AGE));
+        assertThrows(RegistrationException.class,
+                () -> service.register(new User(DEFAULT_LOGIN, TEST_PASSWORD, TEST_AGE)));
+    }
+
+    @Test
+    void register_loginLessThanSixCharacters_notOk() {
```

Comment:
I think it's better to create different tests for different cases(edge-cases).
For example, (pseudo-naming) 6-length-ok, 5-length-not-ok, blank(empty) password-not-ok, etc.
Find other places where you checking multiple cases in one method and create separate ones if needed.